src/main/scala/is/hail/annotations/RegionValueBuilder.scala

```diff
@@ -112,6 +112,10 @@ class RegionValueBuilder(var region: Region) {
     }
   }
 
+  def startTuple(init: Boolean = true): Unit = startStruct(init)
```

Comment:
we shouldn't need this...

Reply:
Even though they are identical, I wanted to make it clear when region value builders are used, what is being added. Otherwise, I think it could be confusing. I think we either need both `startTuple` and `startStruct`,  or only `startStructBase`. Your call which you think is better.

Reply:
there's only one in-memory struct, though, which can be either a TTuple or TStruct. Shall we poll the rest of the group? I could definitely be wrong.

Reply:
I think we should either have start/endStructBase which can handle either or start/endStruct and start/endTuple.  Thinking of both TStruct and TTuple as fundamental types makes the latter more natural and it is slightly safer which is nice.

Reply:
ok, happy to defer. I think of tuples and structs as basically the same fundamental type, but treating them differently is fine too.